packages/salesforce-adapter/src/constants.ts

```diff
@@ -436,3 +436,8 @@ export const SBAA_APPROVAL_CONDITION = 'sbaa__ApprovalCondition__c'
 export const SBAA_APPROVAL_RULE = 'sbaa__ApprovalRule__c'
 
 export const UNLIMITED_INSTANCES_VALUE = -1
+
+// Salesforce Errors
+export const SALESFORCE_ERRORS = {
+  INVALID_CROSS_REFERENCE_KEY: 'sf:INVALID_CROSS_REFERENCE_KEY',
+}
```

Comment:
Nit: no need for comment 🤔 

```suggestion
export const SALESFORCE_ERRORS = {
  INVALID_CROSS_REFERENCE_KEY: 'sf:INVALID_CROSS_REFERENCE_KEY',
} as const
```